packages/twenty-front/src/modules/context-store/hooks/useFindManyRecordsSelectedInContextStore.ts

```diff
@@ -14,12 +13,10 @@ export const useFindManyRecordsSelectedInContextStore = ({
   instanceId?: string;
   limit?: number;
 }) => {
-  const objectMetadataId =
-    useContextStoreCurrentObjectMetadataIdOrThrow(instanceId);
-
-  const { objectMetadataItem } = useObjectMetadataItemById({
-    objectId: objectMetadataId,
-  });
+  const objectMetadataItem = useRecoilComponentValueV2(
```

Comment:
Can't we use useContextStoreObjectMetadataItemOrThrow here ? 

Or else, how can we handle the fact that objectMetadataItem is undefined, let's imagine that this hook renders somewhere `contextStoreCurrentObjectMetadataItemComponentState` isn't initialized yet.